### DISCUSSION TITLE
Zim 0.74.2/OS Windows: Debug log with no date/time stamps

### DISCUSSION THREAD

**Discussion by GHSoCoder** [zim-desktop-wiki/zim-desktop-wiki#2075] [Development]
As mentioned since months Zim is so slow (even the old version 0.6x). I tried to get basic infos from debug log. Why are there no date/time stamps in that file?

**Comment by introt** [comment]
May have been considered too verbose to be enabled by default, and no one has implemented a command line switch or config option to enable those. Since page switches etc are logged, there aren't too many situations where timestamps bring much more to the table - general slowness maybe being an exception, though dedicated profiling tools may be a better fit.

**Comment by roland-ruedenauer** [comment]
I would support the request to add timestamps in the logs as they often transport valuable information.
Regarding concerns about logging to much data, it might be preferrable to have a switch controlling activation of logging (or event the amount of it) in the general settings.
